templates/nsq/0/docker-compose.yml.tpl

```diff
@@ -0,0 +1,43 @@
+nsqlookupd01:
+  image: nsqio/nsq
+  command: /nsqlookupd
+nsqlookupd02:
+  image: nsqio/nsq
+  command: /nsqlookupd
+nsqlookupd03:
+  image: nsqio/nsq
+  command: /nsqlookupd
```

Comment:
Why these 3 distinct `nsqlookupd0N` services if they are same?? Why not just one `nsqlookupd` scaled to 3 and use system discovery to run nsqd and nsqdadmin??

Reply:
we need some static servers in the `command:` line for `nsqadmin` and `nsqd`, scaling wont work

Reply:
You could get dynamic config using system discovery, but no problem at all.